### PR TITLE
Fix/rewind checkpoint id selection

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: zcash_client_sqlite/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+## [0.12.2] - 2024-10-10
```
Comment (str4d) [inline zcash_client_sqlite/CHANGELOG.md:10]:
```suggestion
## [0.12.2] - 2024-10-21
```

**Review by str4d — APPROVED** [review]
utACK a82a60aefb399acf9d3de64b4ecbe14a76464578 after changelog fix

**Comment by str4d** [comment]
Published a82a60aefb399acf9d3de64b4ecbe14a76464578 as `zcash_client_sqlite 0.12.2`.

**Review by daira — COMMENTED** [review]
Post-hoc ACK

**Pull request merged**